docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -271,238 +288,512 @@ Go ahead and review each of the following five files in the pack.
+
+```bash
+aws ecr create-repository --repository-name $REGISTRY_NAME/spectro-packs/archive --region $AWS_DEFAULT_REGION
+```
+
+Next, create the repository to store the Hello Universe pack.
+
+```bash
+aws ecr create-repository --repository-name $REGISTRY_NAME/spectro-packs/archive/$NAME --region $AWS_DEFAULT_REGION
+```
+
+This configuration sets up the required environment and repositories for pushing the Hello Universe pack to your ECR Registry.
+
+</TabItem>
+
+<!-- <TabItem label="Basic" value="Basic_Registry">
+
 <br />
 
-Verify the registry server is accessible from outside the tutorials container by visiting the `/health` endpoint. Access the *https://Your-URL-Here/health* in your host browser. Replace the base URL with the Ngrok URL output you received. You should receive a `{"status":"UP"}` response.
+Once you have an active Harbor registry server, access its domain on your web browser and log in using your Harbor credentials. If you have kept the default credentials, the administrator username and password are **admin** and **Harbor12345**, respectively.
+
+![Screenshot of Harbor login](/tutorials/deploy-pack/registries-and-packs_deploy-pack_harbor-login.png)
+
+
+Next, go to the **Projects** section and click on **New Project**. A project in Harbor contains all repositories of an application. This tutorial uses **spectro-oci-registry** as the project name. Leave the other settings as default and click on **OK**.
+
+![Screenshot of Harbor project](/tutorials/deploy-pack/registries-and-packs_deploy-pack_harbor-project.png)
+
+
+</TabItem> -->
+
+</Tabs>
 
 <br /> 
 
 ### Log in to the Registry Server
-Once the registry server's `/health` endpoint shows `UP` status, the next step is to log in and then push the pack to it. The pack you will push is in the tutorials container. Open another bash session into the tutorials container from your local terminal.  
-<br/>
+
+<Tabs groupId="registry-server">
+
+<TabItem label="Spectro Registry" value="Spectro_Registry">
+
+<br />
+
+Once the `/health` endpoint of the registry server displays an `UP` status, proceed to the authentication step. In a new terminal window, start another bash session in the tutorial container.  
+
 
 ```bash
 docker exec -it tutorialContainer bash
 ```
 
-Log in to the registry server using Ngrok's public URL assigned to you. Issue the command below, but replace the URL with your Ngrok URL. The command below uses these credentials to log in to the registry server: `{username: admin, password: admin}`.
-<br/>
+Log in to the registry server using the Ngrok public URL assigned to you. Issue the following command, replacing the URL with your Ngrok URL. The command below uses these credentials to log in to the registry server: `{username: admin, password: admin}`.
+
 
 ```bash
 spectro registry login  --insecure --default --username admin --password admin \
```

Comment:
What is the purpose of the `--insecure` flag? And is it required? (I know you mention this later on, but might be good to clarify here)

Reply:
Yes it's explained further down in the doc, but I was just saying it's not explained at this point and might a user question.

Reply:
I added a short sentence explaining the flag.